blog/_posts/2018-06-13-accessible-scala.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: blog-detail
+post-type: blog
+by: Guillaume Massé
+title: Introducing Accessible Scala
+---
+
+  Have you found yourself where you had to verbally describe code? For example, while pair-programming
+or talking about work during lunch. Verbally describing code is not an easy task, fortunately for you the
+Scala Center is working on this. [SCP-016]: Accessible Scala, was submitted to provide industry-leading support
```

Comment:
The first two sentences are not appropriate. This project isn't about figuring out how to verbally describe code. It's about accessibility. Can you please change the text to make this clear? As-is, it's inaccurate.

Reply:
Please don't say "industry-leading support," that was also not the goal of this project.